frontend/packages/console-telemetry-plugin/src/listeners/segment.ts

```diff
@@ -111,37 +111,57 @@ export const eventListener: TelemetryEventListener = async (
   properties?: any,
 ) => {
   if (TELEMETRY_DISABLED || !apiKey) {
-    // eslint-disable-next-line no-console
-    console.debug(
-      'console-telemetry-plugin: telemetry disabled - ignoring telemetry event:',
-      eventType,
-      properties,
-    );
-    return;
-  }
-  if (TELEMETRY_DEBUG) {
-    // eslint-disable-next-line no-console
-    console.debug('console-telemetry-plugin: received telemetry event:', eventType, properties);
+    if (TELEMETRY_DEBUG) {
+      // eslint-disable-next-line no-console
+      console.debug(
+        'console-telemetry-plugin: telemetry disabled - ignoring telemetry event:',
+        eventType,
+        properties,
+      );
+    }
     return;
   }
   switch (eventType) {
     case 'identify':
       {
         const { user, ...otherProperties } = properties;
-        const id = user?.metadata?.name;
-        if (id) {
+        const clusterId = otherProperties?.clusterId;
+        const organizationId = otherProperties?.organizationId;
+        const username = user?.metadata?.name;
+        if (username) {
+          let anonymousIdInput: string;
+          if (organizationId) {
+            if (username === 'kubeadmin' || username === 'kube:admin') {
```

Comment:
is `kubeadmin` also reserved?

Reply:
I can remove that check, but I though it's sometimes just `kubeadmin`. When I have done the cluster validation, I will also update this.

Reply:
In our call today you said it's fine to keep it, so I kept it for now.